.github/workflows/autocomment-iss-close.yml

```diff
@@ -0,0 +1,28 @@
+name: Comment on Issue Close
+
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  greet-on-close:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Greet User
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const issue = context.payload.issue;
+            const issueCreator = issue.user.login;
+            const issueNumber = issue.number;
```

Comment:
Is it possible to get the linked PR number here as well? If not possible, I think we can leave that

Reply:
we can get that in PR closure not in issue because when user first raise the issue then raise the PR. So its not possible.

Reply:
Ok, no problem then